frontend/src/components/NotFound.vue

```diff
@@ -1,20 +1,24 @@
 <template>
   <div class="column has-text-centered is-three-fifths-desktop is-three-quarters-tablet is-fullwidth-mobile">
     <div class="box has-background-light content">
-      <template v-if="type">
-        <p class="title is-size-5">
-          <span class="is-capitalized">{{ type }}&nbsp;</span> "&nbsp;{{ componentId }}&nbsp;" &nbsp;not found
-        </p>
-        <template v-if="type === 'model'">
-          <p>Visit the Explore page to select one of the integrated models</p>
-        </template>
-        <template v-else>
-          <p>
-            Probably there is a typo in the {{ type }} identifier in the URL
-            <br>Use the search bar above to find other {{ type }}s
+        <template v-if="type">
+          <p class="title is-size-5">
+            <span class="is-capitalized">{{ type }}&nbsp;</span> "&nbsp;{{ componentId }}&nbsp;" &nbsp;not found
           </p>
+          <template v-if="type === 'model'">
+            <p>Visit the Explore page to select one of the integrated models</p>
+          </template>
+          <template v-else>
+            <p>
+              Probably there is a typo in the {{ type }} identifier in the URL
+              <br>
+              Use the
+              <span v-if="type === 'map'">{{ type }} listing</span>
```

Comment:
For `map listing`, it will be even better to add a helper about how to use map listing.

Reply:
Since this error message is to be displayed right next to the map listing, I didn't think it would be necessary to add a helper. Do you have anything in mind on what would be a good helper message?

Reply:
For the "map listing", I though it means the maplisting in the API. Not sure if other people will also be confused. 
Maybe change the message to "use the list on the left sidebar to find other maps.

Reply:
I though the map listing here was the API endpoint used for the Map Viewer. If so, I'm not able to trigger this alternative message:
```
<span v-if="type === 'map'">{{ type }} listing</span>
```
Could you share a broken url for this case?

Reply:
Thanks for the demo @e0.